### PR TITLE
Updates energy resistance armour values on remaining armours and helmets.

### LINKED ISSUE #47484: Some armors didn't have their energy resistance updated
#47152 missed some armor in code/modules/clothing/spacesuits/syndi.dm, consistency issue, affects some traitor hardsuits
@TooFewSecrets

### PR DESCRIPTION
**Pull request by Firecage**
## About The Pull Request
This PR updates the energy resistance on the armours and helmets which #47152 skipped over. Note, there are a handful of armours and helmets where I equaled the energy value instead of making it 10 more than the laser value, and that was mostly for balance reasons(such as the hardhat Ash Drake Cloak and helmet, and a couple of others.) Also, a few which I assume Secrets had purposely kept their values equal even if their PR didn't say so, I still kept it like that. Such as Riot Armour, Bulletproof Armour, Knight Armour, etc.

This PR also fixes the armor list of the bounty hunter armour.

Fixes #47484 

## Why It's Good For The Game
It updates the remaining armours and helmets to be consistent in their protective values with armours and helmets which were already updates.

## Changelog
:cl:
balance: Updates the energy resistance armour values on several armours and helmets.
/:cl:

### CODE REVIEW

**Comment by Krysonism** [comment]
Do we really need to buff ling and cult in the middle of the antag freeze?

**Comment by Firecage** [comment]
Check the original PR.

**Comment by Firecage** [comment]
This is, odd. Exact same Travis error as someone else, so not just me. Checked my master which I updated, same there, with the light replacer and electropack icons somehow being missing from where they should be. Yet their code files haven't been touched in quite awhile, and their .dmi files haven't been touched in quite awhile either.

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -403,7 +403,7 @@
 	name = "flagellant's robes"
 	desc = "Blood-soaked robes infused with dark magic; allows the user to move at inhuman speeds, but at the cost of increased damage."
 	allowed = list(/obj/item/tome, /obj/item/melee/cultblade)
-	armor = list("melee" = -45, "bullet" = -45, "laser" = -45,"energy" = -45, "bomb" = -45, "bio" = -45, "rad" = -45, "fire" = 0, "acid" = 0)
+	armor = list("melee" = -45, "bullet" = -45, "laser" = -45,"energy" = -55, "bomb" = -45, "bio" = -45, "rad" = -45, "fire" = 0, "acid" = 0)
```
Comment (Michael-Ashfield) [inline code/modules/antagonists/cult/cult_items.dm]: Did you intend to make this armour even weaker to energy? It is already 45 weakness overall which is pretty high.

Comment (Firecage) [reply code/modules/antagonists/cult/cult_items.dm]: Yes, I actually did.

**Pull request merged**